packages/flutter_goldens/lib/flutter_goldens.dart

```diff
@@ -58,38 +86,65 @@ class FlutterGoldenFileComparator implements GoldenFileComparator {
   /// The [goldens] and [defaultComparator] parameters are visible for testing
   /// purposes only.
   static Future<FlutterGoldenFileComparator> fromDefaultComparator({
-    GoldensClient goldens,
+    dynamic goldens,
     LocalFileComparator defaultComparator,
   }) async {
     defaultComparator ??= goldenFileComparator;
 
-    // Prepare the goldens repo.
-    goldens ??= GoldensClient();
-    await goldens.prepare();
+    const Platform platform = LocalPlatform();
+    if (!_testingWithSkiaGold(platform)) {
```

Comment:
With a common interface between the two clients, it might even work well to have a factory constructor that decided which client to instantiate based on `_testingWithSkiaGold(platform)`...

Reply:
You could even have the common interface define things like `auth()`, and have `GoldensClient` just treat that as a no-op - such that maybe you'd be able to have one comparator that operated on an abstract client polymorphically.